schemas/context/deviceidentifiers.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/deviceidentifiers",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Device identifiers",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Condensed, normalized encapsulation of device identifiers.",
+  "definitions": {
+    "deviceidentifiers": {
+      "properties": {
+        "https://ns.adobe.com/experience/ecid": {
```

Comment:
This is a namespace identifier not a item name...

How about `deviceID:experienceCloud`?

Reply:
I think what you have here makes sense. The field name is "ecid" under the ns.adobe.com/experience namespace.

Reply:
I am still confused why we want to use a namespace URL for a property?  It's an incorrect use of NS URLs

Reply:
@lrosenthol - I don't understand your comment here. Field namespaces can be in the form of a URI correct? 
In this case the namespace is "https://ns.adobe.com/experience/" the field name is "ecid". 

@rajindal - Because this is intended to be a standard "xdm" mixin, each of these fields should be standard "xdm" namespaced fields as well. Something like "xdm:ecId", "xdm:gcmId", etc.

Reply:
the changes have been made